### ISSUE TITLE
Windows installer doesn't add .jar file association

### ISSUE THREAD

**Issue by kapela86** [corretto/corretto-8#359]
### Describe the bug
".jar" files are not associated to java.exe

### To Reproduce
Install latest version using JRE MSI installer (1.8.0.322 at the time of writing this).

### Expected behavior
Double clicking ".jar" files should run them like with Oracle Java.

### Platform information
```
Windows 10 x64

openjdk version "1.8.0_322"
OpenJDK Runtime Environment Corretto-8.322.06.1 (build 1.8.0_322-b06)
OpenJDK 64-Bit Server VM Corretto-8.322.06.1 (build 25.322-b06, mixed mode)
```

### Additional context
None

**Comment by rgithubli** [comment]
Reproed: 
- Installed corretto 8 JRE (in the question) on win. Cannot see association between `.jar` and `javaw.exe`. 
- corretto 8 **JDK** can associate `javaw.exe` with `.jar` successfully.

**Comment by rgithubli** [comment]
Hi, we made a fix and it'll be in the next release `8.342` on July 19th. 

Closing this issue. Feel free to reopen if this issue still occurs after the release.
